src/client/Microsoft.Identity.Client/Internal/Requests/AuthenticationRequestParameters.cs

```diff
@@ -47,6 +49,10 @@ internal class AuthenticationRequestParameters
                     ExtraQueryParameters[kvp.Key] = kvp.Value;
                 }
             }
+
+            ClaimsAndClientCapabilities = ClaimsHelper.MergeClaimsAndClientCapabilities(
```

Comment:
Should the ClaimsAndClientCapabilities be renamed to Claims? Do I understand correctly that capabilities are passed in a claims request, so they are a special type of a claim? So in this context normal claims and capabilities are both request claims. Also what if in the future a new type of claim is added, then having a generic Claims collection would be more representative? MergeClaimsAndClientCapabilities and WithClientCapabilities method names, I think, are good because they describe the specific separate collections.

Reply:
This is a valid observation Peter. I was preparing for the fix #1639, where claims and client capabilities are treated differently in that: 

- if claims are specified, we need to bypass the token cache (because cached Access Tokens do not have enough claims)
- but not for client capabilities